berserk/clients/broadcasts.py

```diff
@@ -25,23 +25,32 @@ def create(
         self,
         name: str,
         description: str,
+        auto_leaderboard: bool,
         markdown: str | None = None,
-        official: bool = False,
+        tier: int | None = None,
+        players: Any | None = None,
```

Comment:
Players field should be a typed dict with 4 fields, source_name, display_name, rating and title (both optional). Then would need an internal function to convert them to the string